packages/account/src/utils/secp256_point.cairo

```diff
@@ -1,57 +1,64 @@
 // SPDX-License-Identifier: MIT
-// OpenZeppelin Contracts for Cairo v0.18.0 (account/utils/secp256k1.cairo)
+// OpenZeppelin Contracts for Cairo v0.18.0 (account/utils/secp256_point.cairo)
 
 use core::fmt::{Formatter, Error};
 use starknet::SyscallResultTrait;
 use starknet::secp256_trait::{Secp256Trait, Secp256PointTrait};
-use starknet::secp256k1::Secp256k1Point;
 use starknet::storage_access::StorePacking;
 
-/// Packs a Secp256k1Point into a (felt252, felt252).
+/// Packs a Secp256Point into a (felt252, felt252).
 ///
 /// The packing is done as follows:
+///
 /// - First felt contains x.low (x being the x-coordinate of the point).
 /// - Second felt contains x.high and the parity bit, at the least significant bits (2 * x.high +
 /// parity).
-pub impl Secp256k1PointStorePacking of StorePacking<Secp256k1Point, (felt252, felt252)> {
-    fn pack(value: Secp256k1Point) -> (felt252, felt252) {
+pub impl Secp256PointStorePacking<
+    Secp256Point, +Secp256Trait<Secp256Point>, +Secp256PointTrait<Secp256Point>
+> of StorePacking<Secp256Point, (felt252, felt252)> {
+    fn pack(value: Secp256Point) -> (felt252, felt252) {
         let (x, y) = value.get_coordinates().unwrap_syscall();
 
         let parity = y % 2;
-        let xhigh_and_parity = 2 * x.high.into() + parity.try_into().unwrap();
+        let xhigh_and_parity = 2 * x.high.into() + parity;
 
-        (x.low.into(), xhigh_and_parity)
+        (x.low.into(), xhigh_and_parity.try_into().unwrap())
```

Comment:
`parity` could be represented by felt252 and `x.high` can be also converted to felt with `into` call. This way we could avoid having `try_into` and `unwrap` calls. What do you think?

Reply:
I tried to avoid it, but since y is a u256, to make parity a felt252 we would also need to `.try_into().unwrap()` y % 2. But we can make parity a u128 using y.low % 2 and also remove the `.try_into().unwrap()`. Updated!

Reply:
I was thinking about `let parity: felt252 = y.low % 2 == 0 ? 0 : 1`. But this solution also works since we've got rid of `try_into`!